composer.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "savaslabs/squabble-tests",
```

Comment:
Could we add this to the `require-dev` package in `source/composer.json`, and then place the behat tests in `source/tests/features`?